doc/sphinxext/gallery_generator.py

```diff
@@ -186,6 +214,14 @@ def sphinxtag(self):
     def pagetitle(self):
         return self.docstring.strip().split("\n")[0].strip()
 
+    @property
+    def overlay_description(self):
+        return self.title + (
+            " using {apiname}".format(apiname=self.apiname)
```

Comment:
Important note. The `apiname` property is currently text that is rendered as a link which means that now the hover text has an unreachable link. This is not very problematic for mouse users but breaks keyboard navigation as each card has two focus elements, the card itself and an invisible extra focus that should not be there. If we kept this hover text we would have to fix this.